Sources/xcodeproj/Project/WorkspaceSettings.swift

```diff
@@ -99,13 +97,12 @@ public class WorkspaceSettings: Codable, Equatable, Writable {
     ///
     /// - Parameter path: The path to write to
     /// - Parameter override: True if the content should be overriden if it already exists.
-    /// - Throws: writing error if something goes wrong.
-    public func write(path: Path, override: Bool) throws {
+    public func write(path: Path, override: Bool) {
         let encoder = PropertyListEncoder()
```

Comment:
I've updated all the classes that can be publicly initialized with a path to assume that the developer has checked beforehand whether the path exists or not.

Reply:
Where the result is just a red-item in the open Xcodeproj, I think that's fine, though here I think (like before) writes can `throw` since they are runtime errors rather than programmer errors.